app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: 'user_id'
+  has_many :comments, foreign_key: 'user_id'
+  has_many :likes, foreign_key: 'user_id'
```

Comment:
- [ ] As per the given _ERD_, the _User_ **association** should have _**author_id**_ (_should be different from the name **user**_) referenced as **foreign key** in _Post_, _Comment,_ and _Like_ models. Kindly ensure to improve this aspect, as can be seen below:
```ruby
has_many :posts, foreign_key: 'author_id'
has_many :comments, foreign_key: 'author_id'
has_many :likes, foreign_key: 'author_id'
```

Reply:
Thank you for letting me know. I did it.